src/renderer.rs

```diff
@@ -138,12 +139,35 @@ pub fn page(
                     a.back href="#top" {
                         (arrow_up())
                     }
+                    @if !hide_version_footer {
+                        (version_footer())
+                    }
                 }
             }
         }
     }
 }
 
+// Get version info
+fn get_version_info() -> String {
+    const PROG_NAME: Option<&'static str> = option_env!("CARGO_PKG_NAME");
+    const VERSION: Option<&'static str> = option_env!("CARGO_PKG_VERSION");
+    format!(
+        "{}/{}",
+        PROG_NAME.unwrap_or("miniserve"),
+        VERSION.unwrap_or("unknown")
+    )
+}
+
+// Partial: version footer
+fn version_footer() -> Markup {
+    html! {
+        p style="text-align:center;padding-top:1em" {
```

Comment:
Let's make this a little less visible. I suggest making it smaller and giving it a somewhat succinct color (depending on theme).

The `at_color` in `style.scss` seems like a good candidate to build off of. I suggest make a new variable `footer_color` in every theme, use the respective values of `at_color` and just tone down the contrast a little.